src/main/java/application/services/DefaultMenuService.java

```diff
@@ -0,0 +1,66 @@
+package application.services;
+
+import application.exceptions.NothingSelectedException;
+import application.model.Client;
+import application.model.Dish;
+import application.model.Order;
+import application.repositories.DishesRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component("service")
+public class DefaultMenuService implements MenuService {
+
+    private DishesRepository repository;
+
+
+    @Autowired
+    public DefaultMenuService(DishesRepository repository) {
+        this.repository = repository;
+    }
+
+    @Override
+    public List<Dish> getMenu() {
+        List<Dish> dishList = repository.findAll();
+        return dishList;
+    }
+
+    @Override
+    public List<Dish> getSortedByPriceMenu() {
+        return new ArrayList<>();
+    }
+
+
+    @Override
+    public Dish findByName(String name) {
+        return repository.findByDishNameIgnoreCase(name);
+
+    }
+
+    public List<Dish> performSortedSearch(String sorter) {
```

Comment:
Ok, but if we add a new feature, for example dishRate and we want to perform sorting by rate, we should expand our API? In our case, we can just add 1 procedure to performSortedSerch method. Isn't it useful for scalability of the project?